### PR TITLE
Adding Writing Guidelines Section

### PR DESCRIPTION
**Pull request by Radhika-Saini**
Work on this issue https://github.com/conda-forge/conda-forge.github.io/issues/1680
I have made some changes
Kindly review.

### CODE REVIEW

File: src/user/contributing.rst [hunk 1]
```diff
@@ -49,4 +49,18 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
 5. Submit a `pull request
    <https://help.github.com/articles/about-pull-requests/>`_ to the main repository proposing your changes.
 
+
+.. _Guidelines:
+
+Writing Guidelines
+===========================
```
Comment (kathatherine) [inline src/user/contributing.rst:56]: Make sure the header formatting line is the same length as the header title.

File: src/user/contributing.rst [hunk 2]
```diff
@@ -49,4 +49,18 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
...
+
+Follow these guidelines for contributing or amending conda-forge documentation:
+
+1. For submitting or updating conda-build documentation, follow these rules.
```
Comment (kathatherine) [inline src/user/contributing.rst:60]: Which rules? The rules below, or are you missing a link?

File: src/user/contributing.rst [hunk 3]
```diff
@@ -49,4 +49,18 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
...
+2. Determine who your target audience is, their level of expertise, and how they will use the content.
+3. Match the technical jargon to the audience's degree of understanding. It's preferable to underestimate rather than overestimate your readers' knowledge. Limit the usage of technical phrases to those that the user is likely to encounter. Use a glossary to augment definitions in the text if you need to define a large number of terms.
```
Comment (kathatherine) [inline src/user/contributing.rst:62]: Ooh, "augment" is a great word, but let's make this as accessible as possible. Maybe "supplement" or "enhance"?

**Review by kathatherine — CHANGES_REQUESTED** [review]
This is fantastic. Thanks so much. Just a few edits!

**Comment by jaimergp** [comment]
Superseded by https://github.com/conda-forge/conda-forge.github.io/pull/1723

**Pull request closed without merging**